tests/test_identifier.py

```diff
@@ -44,7 +44,7 @@ def test_identifier_works3():
     out = r.identify("fixtures/file", only_text=False)
     assert (
         "Dogecoin (DOGE) Wallet Address"
-        in out["Regexes"]["file"][1]["Regex Pattern"]["Name"]
+        in out["Regexes"]["file"][2]["Regex Pattern"]["Name"]

```

Comment:
This is scary 👻

Reply:
@piatrashkakanstantinass  thoughts?

Reply:
For context I needed to change this to get the test to pass, I didn't really look into why this test broke though.

Reply:
> This is scary 👻 

Thankfully we have tests, so issues like this get noticed quickly :D

@RackReaver The problem is that pyWhat finds two SSNs inside of the DOGE address. I don't think the regex can get restricted any more, so please add boundaries to the regex (so that it won't match inside of text) and revert this change in tests. :)

Reply:
Done.

Reply:
Thank you! Looks good to me now! 😄